content/go/concepts/loops/loops.md

```diff
@@ -0,0 +1,138 @@
+---
+Title: 'Loops'
+Description: 'There are several different types of loops in Go. All are written using the `for` command using different syntax.'
+Subjects:
+  - 'Code Foundations'
+  - 'Computer Science'
+Tags:
+  - 'Control Flow'
+  - 'Iterators'
+  - 'Loops'
+CatalogContent:
+  - 'learn-go'
+  - 'paths/back-end-engineer-career-path'
+---
+
+There are several different types of loops in Go. All are written using the `for` command using different syntax.
+
+## Three Component Loop
+
+The three component loop syntax is the same as ones used in C or Java. This is what is most often called a `for` loop in other languages.
+
+### Syntax
+
+```pseudo
+for init; condition; post {
+  statements
+}
+```
+
+Here `init` is a statement initalizing an index variable. `condition` is a boolean expression checking the index variable before every loop, telling the loop to execute as long as it's true. And `post` is a statement executed after every loop to increment/decrement the index variable.
+
+### Example
+
+```go
+countdown := 10
+for i := countdown; i > 0; i-- {
+  fmt.Println(i) // Counts down from 10 to 1
+}
+```
+
+## The While Loop
+
+A "while" loop is one where the loop continues to execute as long as some condition remains true. In some languages this is implemented with a keyword `while`, in Go, the "while" loop is implemented with a `for` loop omitting the `init` and `post` statements.
+
+### Syntax
+
+```pseudo
+for condition {
+  statements
+}
+```
+
+The loop will execute as long as `condition` remains true.
+
+### Example
+
+```go
+countdown := 10
+for countdown > 0 {
+  fmt.Println(countdown) // counts down from 10 to 1
+  countdown--
+}
+```
+
+## An Infinite Loop
+
+If the condition statement is omitted along with the post and init statements, the loop will execute indefinitely unless a `break` statement is encountered.
```

Comment:
Should post and init continue to be wrapped in backticks like it has been previously in this entry?
```suggestion
If the condition statement is omitted along with the `post` and `init` statements, the loop will execute indefinitely unless a `break` statement is encountered.
```